build.assets/Dockerfile-centos7

```diff
@@ -1,5 +1,9 @@
 ARG RUST_VERSION
 
+# Create an alias to the assets image. Ref: https://github.com/docker/for-mac/issues/2155
+ARG BUILDARCH
+FROM ghcr.io/gravitational/teleport-buildbox-centos7-assets:teleport13-${BUILDARCH} AS teleport-buildbox-centos7-assets
```

Comment:
Any way to propagate image tag from makefile, e.g. from images.mk? Otherwise we'd have to remember to update this every major release (the "teleport13" part specifically).

Ignore if this suggestion doesn't make sense.

Reply:
I don't think we will need to update the version with every release. We've been using the same compiler since Teleport 4, and I don't think this will change (we may update it at some point, but we won't be updating it with every release). I was thinking about exporting the name just for consistency, but after the major release, this will break every time as the tag won't exist in the repo.

Let's leave it as it is. It's easy to change when needed.